dcs/adapters/outbound/http/api_calls.py

```diff
@@ -24,7 +26,11 @@
 def call_ekss_api(*, secret_id: str, receiver_public_key: str, api_base: str) -> str:
     """Calls EKS to get an envelope for an encrypted file, using the receivers
     public key as well as the id of the file secret."""
-    api_url = f"{api_base}/secrets/{secret_id}/envelopes/{receiver_public_key}"
+
+    receiver_public_key_base64 = base64.urlsafe_b64encode(
+        receiver_public_key.encode()
+    ).decode()
```

Comment:
```suggestion
    receiver_public_key_base64 = base64.urlsafe_b64encode(
        base64.b64decode(receiver_public_key)
    ).decode()
```
Current solution would reencode the already encoded key

Reply:
So the receiver_public_key is already b64 encoded but not in a urlsafe way?

Reply:
pubkey comes from the work order token which should include this as base64 encoded string